src/videotoolbox.cs

```diff
@@ -278,6 +304,22 @@ interface VTCompressionProperties {
 		[Mac (10,13), iOS (11,0), TV (11,0)]
 		[Export ("EncoderId")]
 		string EncoderId { get; set; }
+		
+		[Mac (10,15), iOS (13,0), TV (13,0)]
+		[Export ("TargetQualityForAlpha")]
+		float TargetQualityForAlpha { get; set; }
+		
+		[Mac (10,9), iOS (13,0), TV (13,0)]
+		[Export ("GammaLevel")]
+		float GammaLevel { get; set; }
```

Comment:
```// Read/write, CFNumber (see kCMFormatDescriptionExtension_GammaLevel), Optional```didn't know if I needed ```uint``` or ```float```

Reply:
If the documentation doesn't help, then we usually test it in Objective-C to check the actual behavior (and yes, this is tedious 😒).

As for circumstantial evidence, there's existing code using `kCMFormatDescriptionExtension_GammaLevel `with a `double`: https://github.com/Vidvox/hap-in-avfoundation/blob/master/HapInAVFoundation/AVAssetWriterHapInput.m#L691, so it might not be neither `float` nor `uint`.